crates/tabby/src/serve/mod.rs

```diff
@@ -113,6 +114,7 @@ pub struct ServeArgs {
 
 pub async fn main(config: &Config, args: &ServeArgs) {
     valid_args(args);
+    let mut context = TabbyContext::new();
```

Comment:
Seems the only place using this contex is HealthState - you might simply refer it there?

If the purpose is to have a single instance of Context, you might use lazy_static inside health.rs